Aztec/Classes/TextKit/TextView.swift

```diff
@@ -674,17 +679,19 @@ open class TextView: UITextView {
         }
 
         let isBegginingOfListItem = storage.isStartOfNewLine(atLocation: range.location)
+        guard text == "\n" && beforeString == "\n" && afterString == "\n" && isBegginingOfListItem else {
```

Comment:
I know this is coming from older code, but having the newline character not defined as a constant seems wrong.

We can maybe have a new file for character-constants-of-relevance.